Gemfile

```diff
@@ -50,5 +50,6 @@ end
 group :development do
   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
   # gem "spring"
+  gem 'annotate'
```

Comment:
I think you still need to run `rails g annotate:install` in order for it to make the supposed changes.

Reply:
my bad, thanks for pointing out!